.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: required 
+dist: trusty 
+language: generic 
+compiler:
+  - gcc
+notifications:
+  email:
+    on_success: always
```

Comment:
You may want to turn the `on_success` off...unless you are lonely and like the additional email.